### PR TITLE
[cmake] Fix CMake and xrootd builtin

### PR DESCRIPTION
**Pull request by bellenot**
Fixes https://github.com/root-project/root/issues/16149

### CODE REVIEW

File: builtins/xrootd/CMakeLists.txt [hunk 1]
```diff
@@ -13,11 +13,16 @@ set(XROOTD_VERSION "5.5.2")
 set(XROOTD_PREFIX ${CMAKE_BINARY_DIR})
 message(STATUS "Downloading and building XROOTD version ${XROOTD_VERSION}")
 
+list(REMOVE_ITEM XROOTD_CLIENT_LIBRARIES OpenSSL::SSL)
```
Comment (pcanal) [inline builtins/xrootd/CMakeLists.txt:16]: Is `ssl` not needed or is it the wrong spelling?

Comment (bellenot) [reply builtins/xrootd/CMakeLists.txt:16]: It is needed, but only after `ExternalProject_Add`

**Review by pcanal — APPROVED** [review]

**Comment by vepadulano** [comment]
Thank you @bellenot  :heart: !

**Comment by bellenot** [comment]
> Thank you @bellenot ❤️ !

You're very welcome!

**Comment by pcanal** [comment]
Thanks!

**Pull request merged**